uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/device/settings/SamsungSettings.java

```diff
@@ -217,4 +217,15 @@ public void setPinOnDevice(final String password) {
     public void removePinFromDevice(String pin) {
         //TODO: implement removing PIN for SAMSUNG device.
     }
+
+
+    @Override
+    public void disableAppThroughSettings(@NonNull final String packageName) {
+        //TODO: implement disabling app through settings
```

Comment:
Should we throw UnSupportedException, so the test case will fail with this reason if run on Samsung Device.